crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -1128,7 +1148,17 @@ impl RoomInfo {
 
         if !summary.is_empty() {
             if !summary.heroes.is_empty() {
-                self.summary.heroes_user_ids = summary.heroes.clone();
+                // Parse the user IDs from Ruma.
```

Comment:
I think this comment is outdated now?

Reply:
```suggestion
```

Reply:
My bad, I missed that in the rebase 👍

Reply:
Oh thanks for the update, much appreciated 🙏